webapp/channels/src/components/advanced_create_post/advanced_create_post.tsx

```diff
@@ -1704,6 +1472,14 @@ class AdvancedCreatePost extends React.PureComponent<Props, State> {
                         ),
                         ...(pluginItems || []),
                     ].filter(Boolean)}
+                    codeBlockOnCtrlEnter={this.props.codeBlockOnCtrlEnter}
```

Comment:
I assume this sort of thing is part of your other PR, but were you planning on having more of these into `AdvancedTextEditor` as well? This component already requires a disgusting amount of prop drilling, so adding more is scary even if it's just temporary while we get everything cleaned up 😅

Reply:
So... first step is getting rid of `advanced_create_post` and `advance_create_comment`. In order to do that, we have to move stuff somewhere, and the nearest place is `AdvancedTextEditor`.

Then I would love to take another step, to use "domain specific" components, that do just one thing (handle the draft, handle the uploads, etc...). I have done something similar on mobile, but I would love to give it another twist by using context instead of so much prop drilling.

But to do that... we need to unify everything first.